stage1/gc/gc.go

```diff
@@ -134,6 +135,9 @@ func cleanupV1Cgroups() error {
 
 	b, err := ioutil.ReadFile("subcgroup")
 	if err != nil {
+		if os.IsNotExist(err) {
+			return nil // Probably a failed startup
```

Comment:
If we have a diagnostic logger around at this point, it would be nice logging that the expected file is missing and we just ignore cleaning cgroups.